frontend/src/components/mine/MineHeader.js

```diff
@@ -47,16 +48,18 @@ class MineHeader extends Component {
     });
   }
   
-  openModal(event, mineStatusOptions, onSubmit, title, mine) {
+  openModal(event, mineStatusOptions, mineRegionOptions, onSubmit, title, mine) {
     event.preventDefault();
     const initialValues = {
       "name": mine.mine_detail[0] ? mine.mine_detail[0].mine_name : null,
       "latitude": mine.mine_location[0] ? mine.mine_location[0].latitude : null,
       "longitude": mine.mine_location[0] ? mine.mine_location[0].longitude : null,
       "mine_status": mine.mine_status[0] ? mine.mine_status[0].status_values : null,
+      "major": mine.mine_detail[0] ? mine.mine_detail[0].major : false,
```

Comment:
Looks like we're inconsistent around single vs double-quotes. Do you have a preference?

Reply:
double quotes are preferred throughout.

Single quotes on all imports,

Will work towards removing these inconsistencies.